src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.15.0
+1.15.1-20190823+git10f6ee9458
```

Comment:
FWIW, the make task reported the version as `1.11.1-20190823+git10f6ee9458`. I've fixed this manually, but we may want to take a look at why it calculates the wrong version.

Reply:
I wasn't able to reproduce that. Starting with an updated `master` branch of the PHPC repo:

```
$ cd src/libmongoc/
$ git fetch origin
$ git checkout r1.15
$ cd ../../
$ make libmongoc-version-current 
cd src/libmongoc/ && python build/calc_release_version.py > ../LIBMONGOC_VERSION_CURRENT
$ cat src/LIBMONGOC_VERSION_CURRENT
1.15.1-20190823+git6f73903ae0
```

https://github.com/mongodb/mongo-c-driver/commit/6f73903ae0 points to a commit in the r1.15 branch, but https://github.com/mongodb/mongo-c-driver/commit/10f6ee9458 points to a commit in the master branch (1.16-dev).

The commit hash suggests that you had actually checked out the master branch instead of r1.15, but I'm not sure where "1.11.1" came from. If I repeat the above steps to checkout the master branch and regenerate the version file I end up with `1.16.0-20190823+git10f6ee9458`.

Do you want to attempt to reproduce this?

Reply:
Upon further review, it does look like this PR is incorrectly updating the submodule to https://github.com/mongodb/mongo-c-driver/commit/10f6ee9458.

Reply:
Note, the 1.11.1 might be related to https://jira.mongodb.org/browse/CDRIVER-3315 if the make task is checking out a branch other then master or a release branch.

Reply:
> Upon further review, it does look like this PR is incorrectly updating the submodule to [mongodb/mongo-c-driver@10f6ee9](https://github.com/mongodb/mongo-c-driver/commit/10f6ee9458).

Yep, that was the mistake: I checked out from master instead of the 1.15 release branch. Fixed now 👍